src/config/webpack.config.js

```diff
@@ -492,10 +494,13 @@ module.exports = function(webpackEnv) {
             {
               test: lessRegex,
               exclude: lessModuleRegex,
-              use: getStyleLoaders({
-                importLoaders: 2,
-                sourceMap: isEnvProduction && shouldUseSourceMap,
-              }, 'less-loader'),
+              use: getStyleLoaders(
```

Comment:
this is of course irrelevant, but I assume it was the prettier settings in this repo?